Hi, my name is Bruce Lee

mknijnenberg opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

mknijnenberg committed:
introduction added

mknijnenberg committed:
Bruce Lee

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hi, my name is Bruce Lee
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

mknijnenberg replied:
updated the line

mknijnenberg committed:
updated Information.MD

mknijnenberg committed:
changes favorite course

mknijnenberg committed:
Update introduction.md

mknijnenberg committed:
Update introduction.md

codeschool-kiddo committed:
Added new rule about coffee

mknijnenberg committed:
Update dojo_rules.md

mknijnenberg committed:
Merge pull request #2 from mknijnenberg/new_rules

Too many people are spilling coffee

mknijnenberg committed:
merged

anch 'deadly_skills'

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

mknijnenberg committed:
changed the kill list

mknijnenberg committed:
v1.0.1

mknijnenberg committed:
Added more nonesense and this fixes #3

mknijnenberg committed:
Added more & more nonesense and this fixes #3

mknijnenberg committed:
Merge pull request #5 from mknijnenberg/kill_list

Added more & more nonesense and this fixes #3
